### PR TITLE
Update github team management policies

### PR DESCRIPTION
**Pull request by mxmehl**
This implements the changes discussed in the last TC meeting (#75)

### CODE REVIEW

File: project-handbook/github-management.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ Naming is hard. Within OpenRail, all elements such as repositories must be prepe
 
 Only owners of the OpenRail organisation can create new repositories. This is to prevent clutter and unclear permissions.
 
-Please contact the Chair of the Technical Committee if you need an additional repository, with information about its name and potentially unusual settings.
+Please contact the Technical Committee by its email address if you need an additional repository, with information about its name and potentially unusual settings.
```
Comment (cornelius) [inline project-handbook/github-management.md:13]: We probably should add the email address as a link here.

File: project-handbook/github-management.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ Naming is hard. Within OpenRail, all elements such as repositories must be prepe
...
 
-Note that all repositories in OpenRail must be public. If you need a space for private/sensible information, e.g. for deployment, you must do that elsewhere. However, all information in order to run/deploy your project must in some way be transparent to the public.
+Note that all repositories in OpenRail must be public and only contain Open Source. If you need a space for private/sensible information, e.g. for deployment, you must do that elsewhere. However, all information in order to run/deploy your project must in some way be transparent to the public.
```
Comment (cornelius) [inline project-handbook/github-management.md:15]: Maybe even clearer : "and only contain material licensed under an Open Source license."

**Review by cornelius — APPROVED** [review]
Generally ok, two small things in comments.

File: project-handbook/github-management.md (hunk 2 above)
Comment (mxmehl) [reply project-handbook/github-management.md]: Good idea, implemented.

**Pull request merged**